CardTech.ResearchSite/CardTech.ResearchSite/Pages/Grants.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿@page
+@model CardTech.ResearchSite.Pages.GrantsModel
+@{
+}
+<p>FINDING GRANTS</p>
+<p>
+    Whether you're working on the next great medical breakthrough, developing a thought provoking
+    work of art or wanting to do something good for the community, the Office of Research Development
+    and Strategic Iniatives can help you find funding.
+</p>
+
+<div class="panel-group">
```

Comment:
Let's stay consistent and use the Bootstrap [Accordion](https://getbootstrap.com/docs/4.5/components/collapse/#accordion-example) function. See #31.